src/screens/ConnectScreen.js

```diff
@@ -192,8 +195,22 @@ export default class ConnectScreen extends React.Component {
     };
   }
 
+  onRegionChangeComplete(region) {
+    const vets = this.state.veterans.filter(vet => {
+      return parseFloat(vet.lat) <= (region.latitude + region.latitudeDelta/2) && 
+             parseFloat(vet.lat) >= (region.latitude - region.latitudeDelta/2) && 
+             parseFloat(vet.lng) <= (region.longitude + region.longitudeDelta/2) &&
+             parseFloat(vet.lng) >= (region.longitude - region.longitudeDelta/2)});
+    const pos = this.state.parterOrgs.filter(vet => {
+      return parseFloat(vet.lat) <= region.latitude + region.latitudeDelta/2 && 
+             parseFloat(vet.lat) >= region.latitude - region.latitudeDelta/2 &&
+             parseFloat(vet.lng) <= region.longitude + region.longitudeDelta/2 &&
+             parseFloat(vet.lng) >= region.longitude - region.longitudeDelta/2});
+    this.setState({ region: region, currentPos: pos, currentVets: vets });
+  }
+
   onRegionChange(region) {
-    this.setState({ region: region });
+    //this.setState({ region: region});
```

Comment:
if you comment this out then the region state is not going to be updated upon scrolling the screen so idk if this is exactly what you want